wpeverest/everest-forms
# Tweak - Editable Form name from the form builder title

### All Submissions:






### Changes proposed in this Pull Request:

> QoL Change: Allows editing form name from the form builder.

This PR seeks to add a quality of life change for the evf users, where they can simply use the button, make a form name change, and click SAVE to have it immediately reflect upon the database, instead of going to the settings tab, then form name. Its aim is to be as non intrusive as possible, whilst still enabling saving time for evf users.



~Closes # .~

### How to test the changes in this Pull Request:

- Open the form builder

![image](https://user-images.githubusercontent.com/30156167/71459699-cd580500-27d0-11ea-9dd4-716e389ed610.png)

- Click on edit button

- Make required changes

- Expect the changes reflected in the respective field

![image](https://user-images.githubusercontent.com/30156167/71459817-5ec77700-27d1-11ea-8a26-959190bc7645.png)


### Types of changes:




### Other information:




### Changelog entry

> Tweak - Editable Form name from the form builder title.
